LedService/app/src/main/java/com/bksnake/ledservice/LedControlServiceImpl.java

```diff
@@ -3,15 +3,40 @@
 import android.os.RemoteException;
 import android.util.Log;
 
+import vendor.bksnake.smartledcontroller.V1_0.ILedController;
+
 public class LedControlServiceImpl extends ILedControllerInterface.Stub {
     private static final String LOG_TAG = LedControlService.class.getSimpleName();
```

Comment:
Impl

Reply:
What do you mean under this?

Reply:
Which class name you wont to print?
I expect LedControlServiceImpl.

Reply:
oh, sure It's my mistake

Reply:
@AksonovSergei Fix